src/components/Pricing/AllPlans/index.tsx

```diff
@@ -116,16 +110,7 @@ const PlanSection = ({
 }) => {
     return (
         <div className={`flex flex-col ${className}`}>
-            <div className="flex">
-                <Icon className="opacity-30 w-[36px] mr-2" />
-                <div>
-                    <h3 className="m-0 text-lg">{title}</h3>
-                    <p className="m-0 text-black/50 font-medium text-[14px]">{subtitle}</p>
-                </div>
-            </div>
-            <ul
-                className={`grow list-none grid m-0 p-0 sm:gap-x-6 sm:gap-y-0 gap-y-6 mt-5 pt-9 border-gray-accent-light border-dashed border-t sm:grid-cols-${plans.length}`}
-            >
+            <ul className={`grow list-none grid m-0 p-0 sm:gap-x-6 sm:gap-y-0 gap-y-6 sm:grid-cols-${plans.length}`}>
```

Comment:
The value of `plans.length` looks like it will always be 1, so can we remove this and the Tailwind safelist additions?

Reply:
This version will likely be deleted so I don't think we need to optimize it.

Reply:
Not a huge deal! Mainly just hate adding to `safelist.txt` and I _think_ this is the only reason for those additions.

Reply:
What does `safelist.txt` do in practice? 😄